Command/SystemInfoDumpCommand.php

```diff
@@ -66,43 +81,34 @@ protected function configure()
      * @param $input InputInterface
      * @param $output OutputInterface
      */
-    protected function execute(InputInterface $input, OutputInterface $output)
-    {
-        if ($input->getOption('list-info-collectors')) {
-            $output->writeln('Available info collectors:', true);
-            foreach ($this->registry->getIdentifiers() as $identifier) {
-                $output->writeln("  $identifier", true);
-            }
-        } else if ($identifiers = $input->getArgument('info-collectors')) {
-            foreach ($identifiers as $identifier) {
-                $this->outputInfo(
-                    $this->registry->getItem($identifier),
-                    $output
-                );
-            }
-        } else {
-            foreach ($this->registry->getIdentifiers() as $identifier) {
-                $this->outputInfo($this->registry->getItem($identifier), $output);
-            }
-        }
-    }
+     protected function execute(InputInterface $input, OutputInterface $output)
+     {
+         if ($input->getOption('list-info-collectors')) {
+             $output->writeln('Available info collectors:', true);
+             foreach ($this->registry->getIdentifiers() as $identifier) {
+                 $output->writeln("  $identifier", true);
+             }
+         } else {
```

Comment:
you can do a return after foreach above and get rid of the else indentation for the remaining code